modules/claims_api/app/controllers/claims_api/application_controller.rb

```diff
@@ -51,6 +51,15 @@ def v0?
       request.env['PATH_INFO'].downcase.include?('v0')
     end
 
+    def check_source_user
+      if !header('X-VA-User') && request.headers['X-Consumer-Username']
```

Comment:
why not just use the `v0` based conditional, we already use it for loa, feel like that might simplify and provide an opportunity to refactor with the other v0 headers.

Also could you add a test?

Reply:
Yes I'll include a test, I want to keep loa seperate from source user but both do fall under the V0 if statement above so that it stays within the claims v0 scope.

Reply:
So I couldn't figure out a way to do a test for the sentry entry but I did add the X-Consumer-Username validation back on just set it only to work under V0 in this case. Updated the test accordingly.